config/style.yml

```diff
@@ -131,12 +131,6 @@ Style/Documentation:
   Description: 'Document classes and non-namespace modules.'
   Enabled: false
 
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
-Style/GlobalVars:
-  Description: 'Do not introduce global variables.'
-  Enabled: false
```

Comment:
I assume we end up with this being enabled by removing this? which seems a good thing to me.

Reply:
Yes, that's right. I've done so many now I've stopped mentioning this.